src/webapi/touch.rs

```diff
@@ -0,0 +1,215 @@
+use webapi::event_target::EventTarget;
+use webcore::try_from::{
+    TryFrom,
+    TryInto,
+};
+use webcore::value::{
+    ConversionError,
+    Reference,
+    Value,
+};
+
+/// The Touch interface represents a single contact point on a touch-sensitive
+/// device. The contact point is commonly a finger or stylus and the device may
+/// be a touchscreen or trackpad.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch)
+#[derive(Clone, Debug, Eq, PartialEq, ReferenceType)]
+#[reference(instance_of = "Touch")]
+pub struct Touch( Reference );
+
+impl Touch {
+
+    /// Returns a unique identifier for this Touch object. A given touch point
+    /// (say, by a finger) will have the same identifier for the duration of
+    /// its movement around the surface. This lets you ensure that you're
+    /// tracking the same touch all the time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch/identifier)
+    #[inline]
+    pub fn identifier(&self) -> i64 {
```

Comment:
Why `i64`? (: The specs say that it's `long`, which is supposed to be `i32`, so is the `i64` here by mistake or is there an actual reason for it?